complete.go

```diff
@@ -280,15 +280,10 @@ func matchFile(s string) (matches []string, longest []rune) {
 
 	for _, f := range files {
 		name := filepath.Join(dir, f.Name())
-		f, err := os.Stat(name)
+		f, err := os.Lstat(name)
 		if err != nil {
-			fl, err := os.Lstat(name)
```

Comment:
I'm not sure why this was doing a stat and an lstat. Maybe the stat is for the isDir check later that adds a trailing path separator. However on my Linux machine lstat on a symlink that points to a directory returns a FileInfo with the directory mode bit set, and this behavior is required by POSIX.1-2008.